notebooks/py_scripts/04-Automate_Optional.py

```diff
@@ -91,35 +99,44 @@
 train.ModelTraining.populate()
 
 # %% [markdown]
-# 4. Add this model to the `Model` table and evaluate.
+# 5. Add this model to the `Model` table and evaluate.
 #    - Include a user-friendly `model_name`
 #    - Include the relative path for the project's `config.yaml`
 #    - Add `shuffle` and `trainingsetindex`
+#    - `insert_new_model` will prompt before inserting, but this can be skipped with `prompt=False`
 
 # %%
 model.Model.insert_new_model(model_name='OpenField-5',dlc_config=config_path,
-                             shuffle=1,trainingsetindex=0, paramset_idx=1,
+                             shuffle=1,trainingsetindex=0, paramset_idx=1, 
+                             prompt=True, # True is the default behavior
                              model_description='Open field model trained 5 iterations')
 model.ModelEvaluation.populate()
 
 # %% [markdown]
-# 5. Add a pose estimation task, and launch pose estimation.
+# 6. Auto-populate the `RecordingInfo` table with relevant fields.
+
+# %%
+model.RecordingInfo.populate()
```

Comment:
For the `04-automate` notebook, let's wrap all of the populate statements within a `process.py` script.